HW_Python3/Expl1.py

```diff
@@ -0,0 +1,24 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A. 
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+# В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). 
+# Последняя строка содержит число X.
+
+# *Пример:*
+
+# 5
+#     7 -2 3 5 1
+#     3
+#     -> 1
+
+
+n = input("Введите элементы массива через пробел: ")
+array = n.split()
```

Comment:
Это можно объединить
```suggestion
array = input("Введите элементы массива через пробел: ").split()
```